app/lib/features/space/sheets/link_room_sheet.dart

```diff
@@ -383,6 +383,7 @@ class _LinkRoomPageConsumerState extends ConsumerState<LinkRoomPage> {
         for (final roomId in currentRooms) {
           update.addRoom(roomId);
         }
+        //FIXME : Below is not working at all.
         await room.setJoinRule(update);
```

Comment:
@gnunicorn 
Based on my testing, `await room.setJoinRule(update);` is not working. Can you test it once from your side?

Reply:
added and pushed an integration test proofing this works as expected. Unless you can tell me a specific case where this doesn't I'd mark this resolved. Only note is that this isn't instantaneous but needs a sync, so it might take a second...

Reply:
Ok thanks